packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -102,6 +102,9 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
 
   /** Allow focus to move to root */
   allowFocusRoot?: boolean;
+
+  /** Allows tab key to be handled */
+  allowTabKey?: boolean;
```

Comment:
if allowTabKey is true the user will no longer be able to move out of a focusZone (unless it is being handled outside of the FocusZone), correct? That should be noted on the prop so developers know the implications of setting this prop to true.